Sources/Exporters/Jaeger/Adapter.swift

```diff
@@ -109,6 +114,18 @@ final class Adapter {
         case let .double(value):
             vType = .double
             vDouble = value
+        case .stringArray(let value):
```

Comment:
I wonder if array of arrays supported?

Reply:
No, only arrays of previous possible attribute values: string, bool and numeric (double or int). Should array of arrays be supported?